lib/ansible/modules/crypto/openssh_cert.py

```diff
@@ -0,0 +1,542 @@
+               where timespec can be an integer + [w | d | h | m | s] (e.g. +32w1d2h).
+               Note that if using relative time this module is NOT idempotent."
+    valid_to:
+        required: true
+        type: str
+        description:
+            - "The point in time the certificate is valid to. Time can be specified either as relative time or as absolute timestamp.
+               Time will always be interpreted as UTC. Valid formats are: [+-]timespec | YYYY:MM:DD | YYYY:MM:DD:HH:MM:SS | YYYY:MM:DD HH:MM:SS | forever
+               where timespec can be an integer + [w | d | h | m | s] (e.g. +32w1d2h).
+               Note that if using relative time this module is NOT idempotent."
+    valid_at:
+        required: false
+        type: str
+        description:
+            - "Check if the certificate is valid at a certain point in time. If it is not the certificate will be regenerated.
+               Time will always be interpreted as UTC. Mainly to be used with relative timespec for valid_from and / or valid_to.
+               Note that if using relative time this module is NOT idempotent."
+    principals:
+        required: false
+        type: list
+        description:
+            - "Certificates may be limited to be valid for a set of principal (user/host) names.
+              By default, generated certificates are valid for all users or hosts."
+    options:
+        required: false
+        type: list
+        description:
+            - "Specify certificate options when signing a key. The option that are valid for user certificates are:"
+            - "C(clear): Clear all enabled permissions.  This is useful for clearing the default set of permissions so permissions may be added individually."
+            - "C(force-command=command): Forces the execution of command instead of any shell or
+               command specified by the user when the certificate is used for authentication."
+            - "C(no-agent-forwarding): Disable ssh-agent forwarding (permitted by default)."
+            - "C(no-port-forwarding): Disable port forwarding (permitted by default)."
+            - "C(no-pty Disable): PTY allocation (permitted by default)."
+            - "C(no-user-rc): Disable execution of ~/.ssh/rc by sshd (permitted by default)."
+            - "C(no-x11-forwarding): Disable X11 forwarding (permitted by default)"
+            - "C(permit-agent-forwarding): Allows ssh-agent forwarding."
+            - "C(permit-port-forwarding): Allows port forwarding."
+            - "C(permit-pty): Allows PTY allocation."
+            - "C(permit-user-rc): Allows execution of ~/.ssh/rc by sshd."
+            - "C(permit-x11-forwarding): Allows X11 forwarding."
+            - "C(source-address=address_list): Restrict the source addresses from which the certificate is considered valid.
+               The address_list is a comma-separated list of one or more address/netmask pairs in CIDR format."
```

Comment:
```suggestion
               The C(address_list) is a comma-separated list of one or more address/netmask pairs in CIDR format."
```

Reply:
@thomwiggers you also seem to like the "suggested change" feature :D